apps/deploy-web/src/components/layout/Layout.tsx

```diff
@@ -102,42 +87,58 @@ const LayoutApp: React.FunctionComponent<Props> = ({ children, isLoading, isUsin
   };
 
   return (
-    <>
-      <WelcomeModal open={isShowingWelcome} onClose={onWelcomeClose} />
-
-      <div className="h-full">
-        <div className="h-full w-full" style={{ marginTop: `${ACCOUNT_BAR_HEIGHT}px` }}>
-          <div className="h-full">
-            <Nav isMobileOpen={isMobileOpen} handleDrawerToggle={handleDrawerToggle} />
-
-            <div className="block h-full w-full flex-grow rounded-none md:flex">
-              <Sidebar onOpenMenuClick={onOpenMenuClick} isNavOpen={isNavOpen} handleDrawerToggle={handleDrawerToggle} isMobileOpen={isMobileOpen} />
-
-              <div
-                className={cn("ease ml-0 h-full flex-grow transition-[margin-left] duration-300", {
-                  ["md:ml-[240px]"]: isNavOpen,
-                  ["md:ml-[57px]"]: !isNavOpen
-                })}
-              >
-                {isLoading !== undefined && <LinearLoadingSkeleton isLoading={isLoading} />}
-
-                <ErrorBoundary FallbackComponent={ErrorFallback}>
-                  {!isUsingSettings || isSettingsInit ? (
-                    !isUsingWallet || isWalletLoaded ? (
-                      <div className={cn({ ["container pb-8 pt-4"]: !disableContainer }, containerClassName)}>{children}</div>
-                    ) : (
-                      <Loading text="Loading wallet..." />
-                    )
+    <div className="h-full">
+      {/* Banner for Free Trial and Credit Card payments */}
```

Comment:
it says the same with the text below? is this comment needed? )

Reply:
Guess not